kyo-bench/src/main/scala/kyo/bench/MtlBench.scala

```diff
@@ -21,13 +21,13 @@ class MtlBench extends BaseBench:
                     _    <- Var.update((state: State) => state.copy(value = state.value + 1))
                 yield ()
             )
-        Var.run(State(2))(
-            Emit.run(
-                Env.run(EnvValue("config"))(
-                    testKyo.andThen(Var.get[State])
-                )
-            )
-        )
+
+        testKyo.handle(
+            Var.run(State(2)),
+            Emit.run(_),
+            Env.run(EnvValue("config")),
+            Abort.run
+        ).eval
```

Comment:
I did some initial performance testing and noticed that Kyo was essentially a no-op in this bench because it was missing `Abort.run` and `eval` 🤦

Reply:
You can't be the performance of no-op!